### ISSUE TITLE
Separate working directories for intercepts

### ISSUE THREAD

**Issue by simonklb** [telepresenceio/telepresence#2818]
**Please describe your use case / problem.**

The working directory for all intercepts seems to be based on the current working directory from where you run `telepresence helm install`. It would be great if the working directory for each intercept is set to the current working directory from where you execute the intercept command.

Current state:

```
$ cd ./a
$ telepresence helm install
$ cd ../b
$ telepresence intercept service-b --port 1234:1234 -- pwd
/home/simon/a
$ cd ../c
$ telepresence intercept service-c --port 1234:1234 -- pwd
/home/simon/a
```

**Describe the solution you'd like**

```
$ cd ./a
$ telepresence helm install
$ cd ../b
$ telepresence intercept service-b --port 1234:1234 -- pwd
/home/simon/b
$ cd ../c
$ telepresence intercept service-c --port 1234:1234 -- pwd
/home/simon/c
```

**Describe alternatives you've considered**

Right now you have to execute the command you want to intercept with using an absolute path. It works but it's inconvenient.

**Versions (please complete the following information)**

```
$ telepresence version
Client: v2.8.2 (api v3)
Root Daemon: v2.8.2 (api v3)
User Daemon: v2.8.2 (api v3)
```

**Comment by Xavientois** [comment]
We are also experiencing this behaviour when we run `telepresence connect` before intercepting. Previously, when we were on version `2.5.6`, the `telepresence intercept` command would run in the directory where it was executed. Any idea which version added the current behaviour?

**Comment by simonklb** [comment]
This seems to be fixed in 2.9.x. Thanks!
